app/pages/search/controls/SearchControlsContainer.js

```diff
@@ -121,6 +121,7 @@ class UnconnectedSearchControlsContainer extends Component {
     }
 
     this.handleFiltersChange(emptyFilters);
+    this.handleSearch(emptyFilters);
```

Comment:
Can you write unit test for this, something as 
`when reset button was click, trigger handleSearch`

Reply:
It already failed with couple of manual testing. I will find the cause and fix the bug first.

Reply:
But still, write unit tests for it

Reply:
I meant the reset button was not working as intended with couple of manual checks and wanted to fix it first. Missed to write it clearly before. Sorry, My bad!